guides/WSL2_SETUP.md

```
@@ -0,0 +1,603 @@
+
+### Step 1: Verify WSLg Installation
+
+WSLg comes pre-installed with Windows 11. Verify the installation:
+
+```bash
+# Check WSLg version
+ls /mnt/wslg/versions.txt && cat /mnt/wslg/versions.txt
+```
+
+Expected output should show WSLg version (e.g., v1.0.71 or later):
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)